app/src/main/java/co/nimblehq/alpine/sample/CameraCaptureActivity.kt

```diff
@@ -52,6 +55,11 @@ class CameraCaptureActivity : ComponentActivity() {
         outputDirectory = getMediaOutputDirectory()
     }
 
+    override fun onDestroy() {
```

Comment:
Should we handle something when `onResume`/`onPause` as well 🤔

Reply:
Based on [this codelab](https://developer.android.com/codelabs/camerax-getting-started#1) It looks like we only need to handle `onDestroy` 🤔

Reply:
Regarding this [comment](https://github.com/nimblehq/alpine/pull/30#issuecomment-1259085208) Not sure we should handle them when `onResume/onPause` because It seems like it lives capturing only when initialize to enter on `CameraCaptureActivity`